blog/_posts/2017-11-10-linux-foundation-OSSummit.md

```diff
@@ -0,0 +1,41 @@
+---
+title: The Linux Foundation Open Source Summit 🇪🇺
+layout: post
+created_at: Wed Nov 11 2017
+permalink: blog/2017-11-15-KaradelaMarck-Linux-Foundation-OSSummit-Europe
```

Comment:
Hi @MarckK. Thanks for the PR. Can you please fix the permalink here? Permalink and file name should be the same.

Reply:
Sure :) I just had an earlier date in the file name so I could render the post locally.